iree/compiler/Dialect/VM/Conversion/VMToEmitC/ConvertVMToEmitC.cpp

```diff
@@ -1359,7 +1494,8 @@ class ConvertVMToEmitCPass
 
     // Control flow ops
     target.addLegalOp<IREE::VM::BranchOp>();
-    target.addLegalOp<IREE::VM::CallOp>();
+    // TODO(simon-camp): Only internal calls should be illeagal for now.
```

Comment:
illegal*
could you make this dynamically legal and check that the call is internal or not? (I'm guessing the todo here is for imported calls?)

Reply:
I dropped that line, so all call ops are illegal and imported calls should fail during the conversion.